contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -677,6 +685,11 @@ class Resource extends mix(APIResource, IndexedDBResource) {
   }
 }
 
+export const Session = new IndexedDBResource({
+  tableName: TABLE_NAMES.SESSION,
+  uuid: false,
```

Comment:
To address your suggestion below, can add:
```
idField: 'constantId',
```
here - and then just query by the constant value you set for users when you set to the table.